ts/Stock/StockToolsGui.ts

```diff
@@ -1223,6 +1227,34 @@ class Toolbar {
             }
         });
     }
+    /**
+     * Check if any of the price indicators are enabled.
+     * @private
+     * @function bindingsUtils.isLastPriceEnabled
+     *
+     * @param {array} series
+     *        Array of series.
+     *
+     * @return {string}
+     *         Tells which indicator is enabled.
+     */
+    public checkIfPriceIndicatorEnabled = function (series: SeriesOptions[]): string {
+        let priceIndicatorEnabled: string = '';
+
+        series.forEach(function (serie): void {
+            const lastVisiblePrice = serie.lastVisiblePrice && serie.lastVisiblePrice.enabled,
+                lastPrice = serie.lastPrice && serie.lastPrice.enabled;
+
+            if (lastVisiblePrice || lastPrice) {
+                priceIndicatorEnabled = 'atLeastOneEnabled';
+            }
+
+            if (!lastVisiblePrice && !lastPrice) {
+                priceIndicatorEnabled = 'bothDisabled';
```

Comment:
Having the boolean variable stored as a string seems a bit strange for me. A better way would be creating some kind of enum, where all options are available, which would prevent us from making some typos. If there are only 2 possibilities, the obvious choice seems `boolean`.